app.py

```diff
@@ -368,6 +369,16 @@ def parse():
     username = current_user()['username'] or None
     exception = None
 
+    user = DATABASE.user_by_username(username)
+
+    # If user has no  test_group field, create and save it into database
+    if "test_group" not in user.keys():
+        test_group = alternative_error_messages.select_users_test_group()
+        DATABASE.update_user(username, {"test_group": test_group})
```

Comment:
I'm a bit hesitant to update the user table. What if we decide to no longer use this option, or want to change the group distribution? Start over? This user table is the core of our app and is hard to clean-up or alter later on. It might be nice to use an additional table that stores this data. Simply like a `username - test group` pair. This also allows us to increase the amount of groups (for example) later on.